client/go/cmd/test.go

```diff
@@ -0,0 +1,357 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+// vespa test command
+// Author: jonmv
+
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/vespa-engine/vespa/client/go/util"
+	"github.com/vespa-engine/vespa/client/go/vespa"
+	"io"
+	"io/ioutil"
+	"math"
+	"net/http"
+	"net/url"
+	"os"
+	"path"
+	"strings"
+	"time"
+)
+
+func init() {
+	rootCmd.AddCommand(testCmd)
+}
+
+var testCmd = &cobra.Command{
+	Use:   "test [tests directory or test file]",
+	Short: "Run a test suite, or a single test",
+	Long: `Run a test suite, or a single test
+
+Runs all JSON test files in the specified directory, or the single JSON
+test file specified.
+
+If no directory or file is specified, the working directory is used instead.`,
+	Example: `$ vespa test src/test/application/tests/system-test
+$ vespa test src/test/application/tests/system-test/feed-and-query.json`,
+	Args:              cobra.MaximumNArgs(1),
+	DisableAutoGenTag: true,
+	Run: func(cmd *cobra.Command, args []string) {
+		target := getTarget()
+		testPath := "."
+		if len(args) > 0 {
+			testPath = args[0]
+		}
+		if count, failed := runTests(testPath, target); len(failed) != 0 {
+			_, _ = fmt.Fprintf(stdout, "\nFailed %d of %d tests:\n", len(failed), count)
```

Comment:
Remove the unnecessary `_, _ =`. Same for all the other occurrences below.

Reply:
IntelliJ complains when these aren't there, but ok :)